ansible/group_vars/vagrant.yml

```diff
@@ -6,13 +6,15 @@ app_user: vagrant
 nginx_vhosts:
 - src: vagrant/phpsw_dev.j2
   dest: phpsw_dev
+  enabled: true
 
 ssl_keys:
```

Comment:
I know not specific to this current review, but what are these SSL keys doing?

Reply:
No idea. I'll remove them when considering the SSL part.